target/linux/x86/64/config-6.6

```diff
@@ -151,7 +151,6 @@ CONFIG_DRM_FBDEV_EMULATION=y
 CONFIG_DRM_FBDEV_OVERALLOC=100
 CONFIG_DRM_GEM_SHMEM_HELPER=y
 # CONFIG_DRM_HYPERV is not set
-CONFIG_DRM_I915=y
 CONFIG_DRM_I915_CAPTURE_ERROR=y
 CONFIG_DRM_I915_COMPRESS_ERROR=y
```

Comment:
Please run `make kernel_oldconfig CONFIG_TARGET=subtarget` to update the configuration.

Reply:
I have just tried this, but the result is a bit confusing, many unrelated kconfig options are changed
check the details in the attached [config-6.6.patch](https://github.com/user-attachments/files/16809250/config-6.6.patch)

Reply:
I am trying to clean up first, and then apply i915 related ones

Reply:
Because many of the kconfig iteams are defined in KernelPackage, it is not easy to make kernel_oldconfig, I will focus on the i915 related ones instead